manifests/m/Microsoft/OpenJDK/11/11.0.10.9/Microsoft.OpenJDK.11.yaml

```diff
@@ -0,0 +1,29 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Microsoft.OpenJDK.11
+Publisher: Microsoft
+PackageName: Microsoft OpenJDK 11 Preview
+Author: Microsoft
+ShortDescription: The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft"s new way to collaborate and contribute to the Java ecosystem.
+Moniker: ms-openjdk-11
+Tags:
+- Java
+- OpenJDK
+- "11"
+- LTS
```

Comment:
This package is not LTS. This tag should be removed.

Reply:
Fixed.